app/models/ring.rb

```diff
@@ -0,0 +1,19 @@
+class Ring
+  def initialize(redirections, redirection_to_link)
```

Comment:
> Why pass in redirections?

I over-architected for some dependency-injection future that doesn't exist now and is unlikely to in the future.